oci/loki/documentation.yaml

```diff
@@ -7,6 +7,10 @@ description: >
   effective and easy to operate. It does not index the contents of the logs,
   but rather a set of labels for each log stream.
   Read more on the [official website](https://grafana.com/oss/loki/).
+  
+  Please note that this repository is now holding a ROCK, not a
```

Comment:
Can you please explicitly mention the tags? I.e. the last Dockerfile-based tag and the 1st ROCK tag